i18n/en.json

```diff
@@ -8375,6 +8391,10 @@
     "id": "model.file_info.is_valid.user_id.app_error",
     "translation": "Invalid value for user_id."
   },
+  {
+    "id": "model.github_release_info.is_valid.id.app_error",
+    "translation": "Invalid value for github ID."
+  },
```

Comment:
If any of these are only used in errors you can trick the CI by setting the translation string to `" "` and then that'll be a few less entries that people need to verify.